### PR TITLE
Hook up android_instrumentation_test to test infrastructure

### PR DESCRIPTION
**Pull request by tgummerer**
Summary:
When android_instrumentation_test was introduced, an integration test
was added, but I forgot to hook it up to the test infrastructure.

This caused us to miss that some binary files broke when importing the
pull request from GitHub.

Hook up the integration test to the test infrastructure and fix the
broken binary files.

Test Plan: buck test with and without android devices attached

### CODE REVIEW

**Comment by tgummerer** [comment]
This needs to be applied on top of #506, otherwise the tests don't pass.  Addresses #501, at least for the empty buck-android-support.jar, I haven't looked at the other empty jars @copumpkin mentioned.

**Comment by sdwilsh** [comment]
#506 will be landing soon.  Note to any other Facebook people: we can't use the normal import flow for this because it's broken for binary jars.

**Comment by sdwilsh** [comment]
I've imported this, but making sure it passes CI before landing it.

File: test/com/facebook/buck/android/AndroidInstrumentationTestIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  @Test
+  public void testAndroidInstrumentationTest() throws IOException {
+    tmpFolder.doNotDeleteOnExit();
+    ProjectWorkspace workspace = TestDataHelper.createProjectWorkspaceForScenario(
+        new AndroidBinaryIntegrationTest(),
+        "android_instrumentation_test_integration_test",
+        tmpFolder);
+    workspace.setUp();
+    workspace.runBuckCommand("test").assertSuccess();
```
Comment (sdwilsh) [inline test/com/facebook/buck/android/AndroidInstrumentationTestIntegrationTest.java]: We need to only run when the android sdk is present.  I'll add the appropriate assume statement here.

**Comment by tgummerer** [comment]
Thank you very much for fixing this up and getting it in :)

**Pull request closed without merging**